### ISSUE TITLE
Automaton should sit during /logout and /shutdown

### ISSUE THREAD

**Issue by E5una** [EdenServer/community#5620]
### Checklist

- [ ] I have searched for existing issues for issues like this one. The issue has not been posted.
- [ ] I have provided reproducible steps. (No "as the title says" posts please. ***Please try to state facts only, and avoid assumptions/opinions.***)
- [ ] I have provided my client version in the details. (type /ver into your game window)
- [ ] ***I have provided proof of my claim (era videos, links to era wiki and/or forum posts)*** Lack of proof will slow down fixes and prioritization. Please provide details so that developers' research time is minimized.

### Details

On Eden you can /logout or /shutdown and your automaton will keep fighting. This shouldn’t be the case. Fariska did retail testing this week, the automaton should sit just like it does if you try to rest while it is in combat.

Version 30181205_0

**Comment by KnowOne134** [comment]
so disengage and sit and keep sitting while being beat upon, correct?

**Comment by FariskaEden** [comment]
Yeah that's correct. It just sat in the same spot it was fighting at while being attacked.
